src/test/java/net/liquidpineapple/pang/objects/HookAndRopeTest.java

```diff
@@ -67,6 +67,24 @@ public void hitTopMoveTest() throws Exception {
     assertFalse(Application.getBoard().getCurrentScreen().objectList.contains(rope));
   }
 
+  /**
+   * Tests the behaviour of the move method when Ypos <= MaxY
+   */
+  @Test
+  public void hitTopTest() throws Exception {
+    HookAndRope rope = new HookAndRope(500, 597, player, 0);
+    assertEquals(600, rope.getYpos(), DELTA);
+    rope.move();
+    assertEquals(597, rope.getYpos(), DELTA);
+    rope.doUpdate();
+    assertFalse(Application.getBoard().getCurrentScreen().objectList.contains(rope));
+  }
+
+  @Test
+  public void TestPlayerGetter() {
+    assertEquals(player, hookAndRope.getPlayer());
```

Comment:
I don't think this increases code coverage at all. As the getter is generated bij lombok and the annotation still won't hit like this. Needless to say keeping it in won't hurt.